resources/test_data/csv/float.csv.json

```diff
@@ -2,7 +2,18 @@
     "columns": [
         {
             "name": "a",
+            "nullable": false,
             "type": "float"
         }
-    ]
+    ],
```

Comment:
Hm?

Reply:
I just exported this somewhere, is seems we export the standard CSV config now :D
But I guess this can be removed